packages/shared/src/hooks/post/useBlockPostPanel.ts

```diff
@@ -61,7 +61,7 @@ const getParams = (tags: BlockTagSelection): Params => {
 const ignoredCall = () => Promise.resolve({ successful: true });
 
 export const useBlockPostPanel = (
-  post: Post,
+  post: Post | ReadHistoryPost,
```

Comment:
Since we now have multiple types maybe better to use `Pick` and only have properties that we actually require from post in general.